src/components/place/comment/CommentList.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Avatar,
+  Box,
+  Flex,
+  Image,
+  Table,
+  TableContainer,
+  Tag,
+  Tbody,
+  Td,
+  Text,
+  Tr,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { useState } from 'react';
+
+import { deletePlaceComment, fetchPlaceComment } from '@/api/place';
+import ConfirmModal from '@/components/base/ConfirmModal';
+import Loading from '@/components/base/Loading';
+import MoreMenu from '@/components/base/MoreMenu';
+import { CommentListProps, PlaceCommentType } from '@/types/place';
+import { formatCreatedDateTime } from '@/utils/formatter';
+
+import CommentForm from './CommentForm';
+
+const CommentList = ({ placeId }: CommentListProps) => {
+  const queryClient = useQueryClient();
+  const { data, isLoading, isError } = useQuery<PlaceCommentType>(
+    ['placeCommentList', placeId],
+    () => fetchPlaceComment(10000, placeId)
+  );
+  const { mutateAsync: removePlaceComment } = useMutation(deletePlaceComment);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [commentId, setCommentId] = useState<number>();
+  const [editing, setEditing] = useState({ isEditing: false, commentId: -1 });
+
+  const onCommentDelete = async () => {
+    if (!commentId) return;
+    await removePlaceComment(commentId, {
+      onSuccess: () => {
+        onClose();
+        return queryClient.invalidateQueries(['placeCommentList']);
+      },
+    });
+  };
+
+  if (isLoading) return <Loading></Loading>;
+  if (isError || data.lastId === -1) return <></>;
+
+  return (
+    <>
+      <TableContainer>
+        <Table variant='simple' size='lg'>
+          <Tbody>
+            {data?.locationComments.map((comment) => (
```

Comment:
p4;
```suggestion
            {data?.locationComments.map(({id, profileImage, nickname, memberRole, isEditable}) => (
```
요런식으로 하면 더 좋을 듯 합니다!

Reply:
굳굳! 많아서 고민했는데 계속 comment 반복해서 쓰는 게 더 번거롭더라구용 반영하겠습니다~!